src/status_im2/contexts/chat/composer/utils.cljs

```diff
@@ -181,6 +182,7 @@
   []
   (let [chat-input (rf/sub [:chats/current-chat-input])]
     {:images               (seq (rf/sub [:chats/sending-image]))
+     :link-previews?       (rf/sub [:chat/link-previews?])
```

Comment:
Is this sub just a boolean?

Nitpick: `link-previews` key could be renamed to `links?` for a shorter name, and  I think the word `previews?` here in the composer is redundant (we are still composing, so we know everything is a preview -- images, audio, etc)

Reply:
@OmarBasem, I tried to rename to just links, but then I felt I would need to rename a lot of other stuff to keep naming consistent. I'll revisit your feedback in another PR if you don't mind. Thanks for the suggestion!

Reply:
Sure no problem